front/src/redux/reducers.tsx

```diff
@@ -18,10 +16,7 @@ export const searchParametersReducer = (
     }
 
     case 'ADD_PARAMETER': {
-      state = [
-        ...state,
-        { parameter: '', value: '' },
-      ];
+      state = [...state, { parameter: '', value: '' }];
```

Comment:
State should be immutable.

```suggestion
     return [...state, { parameter: '', value: '' }];
```